CHANGELOG.md

```diff
@@ -16,6 +16,11 @@ of tasks being currently executed.
   - Removed unnecessary fields from mailbox mapping
   - Disable dynamic mapping thanks to a change of the header structure 
   - Read related [upgrade instructions](upgrade-instructions.md)
+  
+### Fixed
+- JAMES-2828 & JAMES-2928 bugs affecting JDBCMailRepository usage with PostgresSQL
```

Comment:
Was thinking... when it's a fix done by a contributor not part of our team, shouldn't we mention his name with a little thanks in the changelog?

Reply:
I don't see how the [JAMES-2928](https://issues.apache.org/jira/browse/JAMES-2928) relates to JDBCMailRepository

Reply:
Good catch, it's JAMES-2929 instead